install/kubernetes/Makefile

```diff
@@ -56,14 +56,35 @@ LOGO_PATH := Documentation/images/logo-solo.svg
 
 all: update-versions $(QUICK_INSTALL) $(EXPERIMENTAL_INSTALL) $(QUICK_HUBBLE_INSTALL) docs
 
+# FIXME change the useDigest to 'true' once images were released by GH actions
 $(QUICK_INSTALL) quick-install: $(shell find cilium/ -type f) update-versions
-	$(QUIET)helm template cilium cilium --namespace=kube-system $(QUICK_OPTIONS) > $(QUICK_INSTALL)
-
+	$(QUIET)helm template cilium cilium --namespace=kube-system \
+     --set image.useDigest=false \
+     --set operator.image.useDigest=false \
+     --set preflight.image.useDigest=false \
+     --set hubble.relay.image.useDigest=false \
+     --set clustermesh.apiserver.image.useDigest=false \
+     $(QUICK_OPTIONS) > $(QUICK_INSTALL)
+
+# FIXME change the useDigest to 'true' once images were released by GH actions
 $(QUICK_HUBBLE_INSTALL) quick-hubble-install: $(shell find cilium/ -type f) update-versions
-	$(QUIET)helm template cilium cilium --namespace=kube-system $(QUICK_HUBBLE_OPTIONS) > $(QUICK_HUBBLE_INSTALL)
-
+	$(QUIET)helm template cilium cilium --namespace=kube-system \
+     --set image.useDigest=false \
+     --set operator.image.useDigest=false \
+     --set preflight.image.useDigest=false \
+     --set hubble.relay.image.useDigest=false \
+     --set clustermesh.apiserver.image.useDigest=false \
```

Comment:
Presumably these new parameters should be no-ops because the default is false.

Reply:
Yes, once we release 1.9, we need to change them to true. (the same does not apply to 1.7 as we already release images based on GH actions)